### ISSUE TITLE
alt="" is equivalent to being a boolean

### ISSUE THREAD

**Issue by chaals** [w3c/html#1243]
There are legitimate uses where the `alt` attribute should be present, but empty.

The code `<img src="something" alt="">` produces the same DOM as `<img src="something" alt>`, so for HTML5 it is effectively the same thing.

We should probably note this in the section where we discuss using blank `alt`.

cc @stevefaulkner @agcolom

**Comment by AmeliaBR** [comment]
See also the WHATWG discussion here: https://github.com/whatwg/html/issues/3502

It basically comes down to a HTML parser issue. The parser treats an attribute with no value as equivalent to an empty string value. That's defined in HTML 5.3 in [section 8.1.2.3. Attributes](https://w3c.github.io/html/syntax.html#elements-attributes).

It isn't the same as a [boolean attribute](https://w3c.github.io/html/infrastructure.html#sec-boolean-attributes), since there are more logical states than simply "absent" or "present".

**Comment by chaals** [comment]
@AmeliaBR said

> It isn't the same as a boolean

Yes, you are right, I was a bit imprecise...
